canopas/react-file-upload
# Implement single file upload management

> PR Description
- Implement a single file upload management component with a preview feature
- Allow uploading of the file by selecting or dragging and dropping the file
- Update `README.md` to mentioned workflow of the project

> Test Steps
- Check the `README.md` on https://github.com/canopas/react-file-upload/tree/single-file-upload#file-management-with-preview
- Checkout this branch and install the dependencies
- Run the dev server by `npm run dev`
- If you face any CSS issues then run `npm run build-css` after running the server
